api/Parser.cpp

```diff
@@ -294,6 +294,11 @@ AllocateEndpoint parseAllocateEndpoint(const nlohmann::json& data)
         allocateEndpoint._data.set(AllocateEndpoint::Data());
     }
 
+    if (data.find("idleTimeout") != data.end())
+    {
+        setIfExists(allocateEndpoint._idleTimeoutSeconds, data, "idleTimeout");
+    }
```

Comment:
`setExists` has name suggest already verify  this: `data.find("idleTimeout") != data.end())`
```suggestion
 setIfExists(allocateEndpoint._idleTimeoutSeconds, data, "idleTimeout");
```